tap_service_titan/streams/accounting.py

```diff
@@ -0,0 +1,555 @@
+    primary_keys: t.ClassVar[list[str]] = ["id"]
+    replication_key: str = "modifiedOn"
+    schema = th.PropertiesList(
+        th.Property("id", th.IntegerType),
+        th.Property("syncStatus", th.StringType),
+        th.Property("referenceNumber", th.StringType),
+        th.Property("date", th.DateTimeType),
+        th.Property("type", th.StringType),
+        th.Property("typeId", th.StringType),
+        th.Property("total", th.StringType),
+        th.Property("unappliedAmount", th.StringType),
+        th.Property("memo", th.StringType),
+        th.Property(
+            "customer",
+            th.ObjectType(
+                th.Property("id", th.IntegerType), th.Property("name", th.StringType)
+            ),
+        ),
+        th.Property(
+            "businessUnit",
+            th.ObjectType(
+                th.Property("id", th.IntegerType), th.Property("name", th.StringType)
+            ),
+        ),
+        th.Property(
+            "batch",
+            th.ObjectType(
+                th.Property("id", th.IntegerType),
+                th.Property("number", th.StringType),
+                th.Property("name", th.StringType),
+            ),
+        ),
+        th.Property("createdBy", th.StringType),
+        th.Property(
+            "generalLedgerAccount",
+            th.ObjectType(
+                th.Property("id", th.IntegerType),
+                th.Property("name", th.StringType),
+                th.Property("number", th.StringType),
+                th.Property("type", th.StringType),
+                th.Property("detailType", th.StringType),
+            ),
+        ),
+        th.Property(
+            "appliedTo",
+            th.ArrayType(
+                th.ObjectType(
+                    th.Property("appliedId", th.IntegerType),
+                    th.Property("appliedTo", th.IntegerType),
+                    th.Property("appliedAmount", th.StringType),
+                    th.Property("appliedOn", th.DateTimeType),
+                    th.Property("appliedBy", th.StringType),
+                    th.Property("appliedToReferenceNumber", th.StringType),
+                )
+            ),
+        ),
+        th.Property(
+            "customFields",
+            th.ArrayType(
+                th.ObjectType(
+                    th.Property("name", th.StringType),
+                    th.Property("value", th.StringType),
+                )
+            ),
+        ),
+        th.Property("authCode", th.StringType),
+        th.Property("checkNumber", th.StringType),
+        th.Property("modifiedOn", th.DateTimeType),
+        th.Property("createdOn", th.DateTimeType),
+        th.Property("active", th.BooleanType),
+    ).to_dict()
+
+    @cached_property
+    def path(self) -> str:
+        """Return the API path for the stream."""
+        return f"/accounting/v2/tenant/{self._tap.config['tenant_id']}/export/payments"
+
+
+class InventoryBillsStream(ServiceTitanExportStream):
```

Comment:
@pnadolny13 good callout, I totally forgot to point out the new stream. Here it is. 

As I add others I may update the prompts to give a little more guidance on replication and primary keys, had to manually set the replication key in this case since it defaulted to `modifiedOn`